src/spicy/spicyz/config.h.in

```diff
@@ -26,21 +26,26 @@ static inline void add_path(std::string& old_path, const path& new_path) {
 }
 } // namespace
 
+static path getEnvOr(const char* name, const char* default_) {
+    if ( auto p = getenv(name); p && *p )
+        return path(p);
+    else
+        return default_;
+}
```

Comment:
This function not only gets an env var, but would also ignore env vars which set to an empty string, e.g., if one sets `ZEEK_SPICY_MODULE_PATH=` `getEnv("ZEEK_SPICY_MODULE_PATH", "DEFAULT")` would return `DEFAULT`. It also always returns a `path`, not e.g., a `std::string`. With that, we should probably rename it, and possibly add a precondition that `default_` is not empty.

```suggestion
static path getEnvPathOr(const char* name, const char* default_) {
    assert(std::strlen(default_) != 0);
    if ( auto p = hilti::rt::getenv(name); p && ! p->empty() )
        return path(*p);
    else
        return default_;
}